--- a/src/Widgets/Window.vala	2024-11-04 09:38:01.214943931 -0800
+++ b/src/Widgets/Window.vala	2024-11-04 09:39:35.025356542 -0800
@@ -113,11 +113,9 @@
             player.volume = value;
         });
 
-        adjust_theme();    // TODO Theme management needs research in flatpak as nonfunctional
         settings.changed.connect( (key) => {
             if (key == "theme-mode") {
                 warning("theme-mode changed");
-                adjust_theme();                     
             }
         });
 
