--- a/src/Widgets/Window.vala	2024-10-29 18:15:52.617122634 -0500
+++ b/src/Widgets/Window.vala	2024-10-29 18:17:19.826125740 -0500
@@ -80,11 +80,9 @@
             player.volume = value;
         });
 
-        adjust_theme();
         settings.changed.connect( (key) => {
             if (key == "theme-mode") {
                 warning("theme-mode changed");
-                adjust_theme();
 
             }
         });
