--- a/src/Widgets/Window.vala
+++ b/src/Widgets/Window.vala
@@ -113,11 +113,9 @@
             player.volume = value;
         });
 
-        adjust_theme();     // FIXME Theme management needs refactoring. adjust-theme calls are removed in a flathub
-        settings.changed.connect( (key) => {
+         settings.changed.connect( (key) => {
             if (key == "theme-mode") {
                 warning("theme-mode changed");
-                adjust_theme();     // FIXME  ...patch so commenting them out now so patch can be removed.
             }
         });
