diff -ur a/src/Widgets/Window.vala b/src/Widgets/Window.vala
--- a/src/Widgets/Window.vala	2024-11-11 10:27:33.000000000 -0800
+++ b/src/Widgets/Window.vala	2024-11-11 12:23:27.465802837 -0800
@@ -114,11 +114,9 @@
             player.volume = value;
         });
 
-        adjust_theme();    // TODO Theme management needs research in flatpak as nonfunctional
         settings.changed.connect( (key) => {
             if (key == "theme-mode") {
                 debug("theme-mode changed");
-                adjust_theme();                     
             }
         });
 
