--- a/Window.vala
+++ b/Window.vala
@@ -113,11 +113,9 @@
             player.volume = value;
         });
 
-        adjust_theme();    // TODO Theme management needs research in flatpak as nonfunctional
         settings.changed.connect( (key) => {
             if (key == "theme-mode") {
                 warning("theme-mode changed");
-                adjust_theme();                     
             }
         });
 
